packages/wrangler/src/__tests__/index.test.ts

```diff
@@ -229,13 +322,27 @@ describe("wrangler", () => {
       const tsconfigJson = JSON.parse(
         fs.readFileSync("./tsconfig.json", "utf-8")
       );
+      // unchanged tsconfig
       expect(tsconfigJson.compilerOptions).toEqual({});
+      const packageJson = JSON.parse(
+        fs.readFileSync("./package.json", "utf-8")
+      );
+      expect(packageJson.devDependencies).toEqual({
+        "@cloudflare/workers-types": expect.any(String),
+      });
     });
 
     it("should not touch an existing package.json in an ancestor directory", async () => {
```

Comment:
This test description is repeated from above. Is that an accident?

Reply:
I dunno, I didn't add either 😅 Did you mean to name it something else?

Reply:
Ahah! Yes. My fault.  This test should be called:

```
should not touch an existing tsconfig.json in an ancestor directory
```